content/blog/2019/01/2019-01-31-ssh-steps-for-jenkins-pipeline.adoc

```diff
@@ -0,0 +1,197 @@
+---
+layout: post
+title: "SSH Steps for Jenkins Pipeline"
+tags:
+- pipeline
+- plugins
+- ssh
+- steps
+author: nrayapati
+---
+
+NOTE: This guest post was originally published on Cerner's Engineering blog
+link:https://engineering.cerner.com/blog/ssh-steps-for-jenkins-pipeline/[here].
+
+
+**Pipeline-as-code** or defining the deployment pipeline through code rather than manual job creation through UI, provides tremendous benefits for teams automating builds and deployment infrastructure across their environments.
+
+image:pipeline/realworld-pipeline-flow.png[alt="Pipeline Flow",width=100%]
+
+_Source of image: link:https://jenkins.io/doc/book/pipeline/[https://jenkins.io/doc/book/pipeline/]_
+
+# Jenkins Pipelines
+
+https://jenkins.io/[Jenkins] is a well-known open source continuous integration and continuous deployment automation tool. With the latest 2.0 release, Jenkins introduced the Workflow plugin that implements Pipeline-as-code. This plugin lets you define delivery pipelines using concise scripts which deal elegantly with jobs involving persistence and asynchrony. 
```

Comment:
```suggestion
https://jenkins.io/[Jenkins] is a well-known open source continuous integration and continuous deployment automation tool. With the latest 2.0 release, Jenkins introduced the Pipeline plugin that implements Pipeline-as-code. This plugin lets you define delivery pipelines using concise scripts which deal elegantly with jobs involving persistence and asynchrony. 
```

Reply:
"Pipeline Plugin" here and below

Reply:
Updated with https://github.com/jenkins-infra/jenkins.io/pull/2076/commits/d9f81aaf32b20d406612501609274d17db41fdae :)